audio-worklet/src/noisegate-script-processor.js

```diff
@@ -203,18 +203,19 @@ class NoiseGate {
    *                               1 - 1/e of its value given a transition from
    *                               0 to 1.
    * @param  {Number} sampleRate The number of samples per second.
-   * @return {Number} Alpha weight governing envelope response.
+   * @return {Number} Weight governing envelope response.
```

Comment:
Not really a "weight" is it?  At least not in the context of all the other weights you have.